cluster/manifests/prometheus/configmap.yaml

```diff
@@ -172,3 +172,8 @@ data:
        - source_labels: [ __name__ ]
          regex: 'reflector.*'
          action: drop
+{{- if ne .ConfigItems.prometheus_remote_write "disabled" }}
+    remote_write:
+    - url: {{ .ConfigItems.prometheus_remote_write }}
+      bearer_token_file: /meta/credentials/remote-write
+{{- end }}
```

Comment:
I learned yesterday from the Prometheus people that we should define two external labels (currently we have `monitor: 'skipper-ingress'` in the top which doesn't seem like the right label anymore). One label should be e.g. `cluster` and the other should be `replica` such that you can de-duplicate the metrics pushed from the two instances. Would make sense to use `POD_NAME` as replica value, but not sure if this can be passed via a flag or how it's easiest to have a different value for the two pods of the StatefulSet.